sdk/evaluation/azure-ai-evaluation/README.md

```diff
@@ -318,6 +338,49 @@ print(outputs)
 ```
 ## Troubleshooting
 
+### General
+
+Azure ML clients raise exceptions defined in [Azure Core][azure_core_readme].
+
+### Logging
+
+This library uses the standard
+[logging][python_logging] library for logging.
+Basic information about HTTP sessions (URLs, headers, etc.) is logged at INFO
+level.
+
+Detailed DEBUG level logging, including request/response bodies and unredacted
+headers, can be enabled on a client with the `logging_enable` argument.
+
+See full SDK logging documentation with examples [here][sdk_logging_docs].
+
 ## Next steps
 
+- View our [samples][evaluation_samples].
+
```

Comment:
- View our [documentation](https://learn.microsoft.com/azure/ai-studio/how-to/develop/evaluate-sdk)